packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
I'm still fairly sure it's supposed to be an error, but yeah who knows if anything gets it right yet. For instance Chrome does throw on

```
function rest(b = a, a) {
console.log(b, a);
}

rest(undefined, 2);
```

but with the `...` it does not, who knows.